fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7119,11 +7121,13 @@ class KeyValueStoreRedwoodUnversioned : public IKeyValueStore {
 			}
 		} else {
 			f = cur.seekLT(keys.end);
+			TEST(f.isReady()); // readRange_impl seekLT acquire lock and wait on disk
+			TEST(!f.isReady()); // readRange_impl seekLT skip lock because data is cached
```

Comment:
you want to include the full condition here, so "!isLocked && !f.isReady()" should be one, and "isLocked || f.isReady()" should be the other